src/Spectre.sln

```diff
@@ -47,22 +47,18 @@ Global
 		{BBF6329D-7BEC-494E-9FC9-B24B12D94EA0}.Release|Any CPU.ActiveCfg = Release|x64
 		{BBF6329D-7BEC-494E-9FC9-B24B12D94EA0}.Release|x64.ActiveCfg = Release|x64
 		{BBF6329D-7BEC-494E-9FC9-B24B12D94EA0}.Release|x64.Build.0 = Release|x64
-		{70C87AF7-3189-4F6C-AC85-93309CF7CFDC}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{70C87AF7-3189-4F6C-AC85-93309CF7CFDC}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{70C87AF7-3189-4F6C-AC85-93309CF7CFDC}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{70C87AF7-3189-4F6C-AC85-93309CF7CFDC}.Debug|x64.Build.0 = Debug|Any CPU
-		{70C87AF7-3189-4F6C-AC85-93309CF7CFDC}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{70C87AF7-3189-4F6C-AC85-93309CF7CFDC}.Release|Any CPU.Build.0 = Release|Any CPU
-		{70C87AF7-3189-4F6C-AC85-93309CF7CFDC}.Release|x64.ActiveCfg = Release|Any CPU
-		{70C87AF7-3189-4F6C-AC85-93309CF7CFDC}.Release|x64.Build.0 = Release|Any CPU
-		{01391E76-3D20-4B4B-B727-2A133B35A144}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{01391E76-3D20-4B4B-B727-2A133B35A144}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{01391E76-3D20-4B4B-B727-2A133B35A144}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{01391E76-3D20-4B4B-B727-2A133B35A144}.Debug|x64.Build.0 = Debug|Any CPU
-		{01391E76-3D20-4B4B-B727-2A133B35A144}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{01391E76-3D20-4B4B-B727-2A133B35A144}.Release|Any CPU.Build.0 = Release|Any CPU
-		{01391E76-3D20-4B4B-B727-2A133B35A144}.Release|x64.ActiveCfg = Release|Any CPU
-		{01391E76-3D20-4B4B-B727-2A133B35A144}.Release|x64.Build.0 = Release|Any CPU
+		{70C87AF7-3189-4F6C-AC85-93309CF7CFDC}.Debug|Any CPU.ActiveCfg = Debug|x64
```

Comment:
I still don't understand, why VS *always* introduces this *Any CPU* platform, each time a project is added. Please delete this.